### PR TITLE
Polish up kafka,nats,activemq,rabbitmq plugins to set peer at comsumer side

### PR DESCRIPTION
**Pull request by pg-yang**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

We  have optimized  MQ Topology analysis and  virtual mq  in https://github.com/apache/skywalking/pull/9911
Let's  fill  `peer `  to  consumer EntrySpan  for all MQ plugins

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Release Notes.
 * Support mannual propagation of tracing context to next operators for webflux.
 * Add MQ_TOPIC and MQ_BROKER tags for RocketMQ consumer's span. 
 * Polish up Pulsar plugins to remove unnecessary dynamic value , set peer at consumer side 
+* Polish up kafka,nats,activemq,rabbitmq plugins to set peer at consumer side
```
Comment (wu-sheng) [inline CHANGES.md:30]:
```suggestion
* Polish Kafka plugin to set peer at the consumer side.
* Polish NATS plugin to set peer at the consumer side.
* Polish ActiveMQ plugin to set peer at the consumer side.
* Polish RabbitMQ plugin to set peer at the consumer side.
```

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
@pg-yang So, I could begin the 8.13.0 release?

**Comment by pg-yang** [comment]
> I could begin the 8.13.0 release?

Yes , I want to optimize `activeMQ` in next release  ,   I'm  reading the doc . OK ?

**Comment by wu-sheng** [comment]
Yes, that is not urgent.

**Pull request merged**